meta-lmp-bsp/recipes-security/optee/optee-os-fio_3.10.0.bbappend

```diff
@@ -1,27 +1,40 @@
 OPTEEMACHINE_apalis-imx6 = "imx-mx6qapalis"
-OPTEEMACHINE_qemuarm64 = "vexpress-qemu_armv8a"
-OPTEEMACHINE_imx6ullevk = "imx-mx6ullevk"
-OPTEEMACHINE_imx8qm-mek = "imx-mx8qmmek"
 OPTEEMACHINE_apalis-imx8 = "imx-mx8qmmek"
+OPTEEMACHINE_imx6ullevk = "imx-mx6ullevk"
 OPTEEMACHINE_imx7ulpea-ucom = "imx-mx7ulpeaucom"
-OPTEEMACHINE_imx8mq-evk = "imx-mx8mqevk"
 OPTEEMACHINE_imx8mm-lpddr4-evk = "imx-mx8mmevk"
+OPTEEMACHINE_imx8mq-evk = "imx-mx8mqevk"
+OPTEEMACHINE_imx8qm-mek = "imx-mx8qmmek"
+OPTEEMACHINE_qemuarm64 = "vexpress-qemu_armv8a"
```

Comment:
Nit, but I would prefer the non functional changes to be separated in another commit, easier to review and revert if needed later on in case of regressions.

Reply:
Yep.  Splitting this into 2 commits.